master/static/srv/notebook-entrypoint.sh

```diff
@@ -15,6 +25,7 @@ if [ "$HOME" = "/" ] ; then
     export HOME
 fi
 
-python3.6 -m pip install -q --user /opt/determined/wheels/determined*.whl
+"$DET_PYTHON_EXECUTABLE" -m pip install -q --user /opt/determined/wheels/determined*.whl
 
-jupyter lab --config /run/determined/workdir/jupyter-conf.py --port=${NOTEBOOK_PORT}
+(cd ${WORKING_DIR} && test -f "${STARTUP_HOOK}" && source "${STARTUP_HOOK}")
```

Comment:
With this in a subshell, some things (like exporting environment variables) won't work.

Reply:
good point; I'll use pushd/popd